aopy/analysis/base.py

```diff
@@ -235,6 +235,45 @@ def calc_task_rel_dims(neural_data, kin_data, conc_proj_data=False):
     else:    
         return task_subspace.T, projected_data
 
+def align_latent_dynamics(La, Lb, return_aligned_dynamics=False):
+    """
+    Aligns latent dynamics using Canonical Correlation Analysis (CCA) and computes pairwise Pearson correlation for both aligned and unaligned dynamics.
+    References: Gallego, J. A., Perich, M. G., Chowdhury, R. H., Solla, S. A. & Miller, L. E. Long-term stability of cortical population dynamics underlying consistent behavior. Nat Neurosci 23, 260–270 (2020).
+
+    Args:
+        La (ndarray): Latent dynamics of Dataset A with shape (m, n_timepoints). Usually first dimension is time, however Juancho's code has it as (m, n_t). Keeping it similar to his paper is easier for the computations below.
+        Lb (ndarray): Latent dynamics of Dataset B with shape (m, n_timepoints).
```

Comment:
I think to make things consistent for aopy we should keep the input data shapes the same other functions with time as the first dimension. Then the first couple lines in the function can transpose them and the transposed variables can be used throughout the rest of the function.

Reply:
Also, should the input data be raw neural data or data projected into a lower dimensional latent space?